### ISSUE TITLE
fallback to shell upon filing wrong password

### ISSUE THREAD

**Issue by pmattern** [lxde/lxqt#777]
Recent lxqt-sudo versions sometimes don't display another GUI password prompt after submitting a wrong password but fall back to the shell.

When this happens there won't be the shell prompt
```
$ lxqt-sudo juffed 
Sorry, try again.
```
that can be seen normally when another GUI prompt gets launched but
```
$ lxqt-sudo juffed 
Sorry, try again.
Password:
```
The password can be submitted on that second prompt and the application in question, here JuffED,  gets launched. If another wrong password is filed sometimes the same shell prompt will be displayed again and sometimes lxqt-sudo does launch another GUI password prompt. It doesn't seem to be predictable why which choice is made.
If an lxqt-sudo invocation like the one above is suffixed with ` &` no input can be made on the shell and lxqt-sudo keeps silently running in the background.

I'm pretty sure that the problem did not exist in earlier versions but regression testing turned out pretty hard as commits older than lxde/lxqt-sudo@be50221 which is affected as well fail to build on recent systems due to the changes regarding the build system introduced by this commit.

I'm not sure about the impact of this problem as I don't know how lxqt-sudo gets launched by regular GUI applications using it as backend which is the normal use case.
But given that the shell is pretty important regarding testing I think this problem should be fixed independently.

**Comment by palinek** [comment]
This sounds realy strange to me... and I'm not able to reproduce it here...

**Comment by pmattern** [comment]
Which versions of the other LXQt components are you running and compiled against which Qt version?

**Comment by palinek** [comment]
Yes, debian testing.... LXQt components from siduction. But for this testing:
- lxqt-sudo ... su_backend branch
- liblxqt ... latest master
- qt ... debian testing version -> Version: 5.4.2+dfsg-5

Also your example with medit works here:
![screenshot](https://cloud.githubusercontent.com/assets/10990929/9137136/f3f58394-3d1b-11e5-9655-0a2b5052c40e.png)

**Comment by palinek** [comment]
@pmattern Is this still valid?
